.github/workflows/scheduled.yml

```diff
@@ -313,6 +313,13 @@ jobs:
           path: velox/_build/debug/velox/functions/prestosql/fuzzer/velox_window_fuzzer_test
           retention-days: "${{ env.RETENTION }}"
 
+      - name: Upload cache fuzzer
+        uses: actions/upload-artifact@v4
+        with:
+          name: cache
```

Comment:
Give it a more relevant name , like cache-fuzzer-artifact

Reply:
I was following the naming style like others in the file. Ex: window, join, aggregation etc. Do you recommend changing this still, may be call it "cache-fuzzer"?